libs/viewer/src/SimpleViewer.cpp

```diff
@@ -1644,7 +1646,17 @@ void SimpleViewer::updateUserInterface() {
     // At this point, all View settings have been modified,
     //  so we can now push them into the Filament View.
     applySettings(mSettings.view, mView);
-    mView->setIblOptions(mSettings.ibl);
+
+    // Set IBL options
+    {        
+        float intensity = mSettings.ibl.iblTintAndIntensity.w;
```

Comment:
Why do we have to store the intensity in a temporary?

Reply:
Fixed in https://github.com/shapr3d/filament/pull/50/commits/e91b7ad0d4ae77461c31886f97b1a2121e7eab6f